packages/ui/src/composites/wui-account-button/index.ts

```diff
@@ -74,7 +75,7 @@ export class WuiAccountButton extends LitElement {
     }
     if (this.balance) {
       const networkElement = this.networkSrc
-        ? html`<wui-image src=${this.networkSrc}></wui-image>`
+        ? html` <wui-image src=${this.networkSrc}></wui-image>`
```

Comment:
This is possibly a result of a Prettier formatting process. Let's remove and check if it's happening again. I generally prefer avoiding empty spaces in renders. 

```suggestion
        ? html`<wui-image src=${this.networkSrc}></wui-image>`
```